resources/assets/lib/click-to-copy.tsx

```diff
@@ -34,7 +33,7 @@ export default class ClickToCopy extends React.Component<Props> {
     }
 
     // copy url to clipboard
-    clipboard.writeText(this.props.value);
+    navigator.clipboard.writeText(this.props.value);
```

Comment:
`navigator.clipboard` requires https (otherwise it becomes `undefined`, I think).
Also noticed all the stuff that comes after this should be running after the promise from `writeText` completes instead.